source/administration/mmctl-cli-tool.rst

```diff
@@ -2976,7 +2976,26 @@ mmctl post list
 mmctl roles
 -----------
 
-This command will be available in a future release.
+**Description**
+
+  Management of user roles.
```

Comment:
```suggestion
  Manage user roles.
```

Reply:
I couldn't decide whether the original matched the changes you made to the style of the descriptions (which read much better!) or whether it should be changed. LMK which is correct?

Reply:
@justinegeffen Your suggestion is a better approach. Thanks!